core-build/gulp-core-build-sass/src/SassTask.ts

```diff
@@ -94,8 +94,8 @@ export class SassTask extends GulpTask<ISassTaskConfig> {
     super(
       'sass',
       {
-        preamble: '/* tslint:disable */',
-        postamble: '/* tslint:enable */',
+        preamble: '/* eslint-disable */',
```

Comment:
Would it make sense to emit both comments, to be compatible with both ESLint and TSLint? 

I still encounter a lot of projects that have not migrated to ESLint yet.

Reply:
I think for those wanting to use this package directly, i.e. without the depending on the @rushstack build tools, emitting both comments would be helpful. That way they can have their own implementation of either TSLint or ESLint and still be able to use this gulp task without additional configuration.

Reply:
@sadsa Could you update your PR with this fix?